lumigator/frontend/src/helpers/lumigator-axios-instance.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
+import { getAxiosError } from './getAxiosError'
+
+export const lumigatorApiAxiosInstance = axios.create({
+  baseURL: '/api/v1/', // Set to relative path to the backend
+  headers: {
+    'Content-Type': 'application/json',
+  },
+})
+
+lumigatorApiAxiosInstance.interceptors.response.use(
+  (response) => response,
+  (error) => {
+    console.error(getAxiosError(error))
+
+    throw error
+  },
+)
```

Comment:
this is a global interceptor, so any error handling logic that needs to be applied for all requests can go here, in this case we're logging the error message